blocks/card-social-comment/src/CardSocialComment.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { ArrowForward, TurnRight, StarBorder, MoreHoriz } from "@mui/icons-material"
```

Comment:
For icons, let's use plain SVG so that the copy & paste experience is at best.

Pick an icon from [heroicons.com](https://heroicons.com/) or [lucide](https://lucide.dev/icons/) or any site that you prefer.

Then wrap it with `SvgIcon` from Joy UI:

```js
import SvgIcon from '@mui/joy/SvgIcon';

<SvgIcon>
  <svg>…</svg>
</SvgIcon>
```